### DISCUSSION TITLE
Can I see what version of etherpad an instance is using?

### DISCUSSION THREAD

**Discussion by jaimet** [ether/etherpad-lite#6714] [Q&A] [answered]
For example, I go to:

https://e.opnxng.com

(as a user). How do I find out what version of etherpad this particular instance is using?

**Comment by SamTV12345** [comment, marked as answer]
> For example, I go to:
> 
> 
> 
> https://e.opnxng.com
> 
> 
> 
> (as a user). How do I find out what version of etherpad this particular instance is using? 

You can use scanner.etherpad.org to find out which instance uses which version ☺️
